shared/testgrid/testgrid.go

```diff
@@ -62,6 +62,11 @@ func GetArtifactsDir() string {
 	dir := os.Getenv("ARTIFACTS")
 	if dir == "" {
 		log.Printf("Env variable ARTIFACTS not set. Using './artifacts' instead.")
+		err := os.Mkdir("artifacts", 0777)
+		if err != nil {
```

Comment:
It should not fail if the directory already exists.

Reply:
it does not. You can verify by running the test multiple times.